kasa/cli.py

```diff
@@ -1244,14 +1238,18 @@ def _print_features(dev):
 
     feat = dev.features[name]
 
+    _echo_features(
+        {name: feat}, "\n[bold]== Feature Current Value ==[/bold]", indent=""
+    )
```

Comment:
Maybe we should avoid using `_echo_features` for a single feature, and just do something like this (if we even want to display the current setting at all?):
```
echo(f"Changing {feat.name} from {feat.value} to {value}")
await dev.update()
echo("New state: {feat.value}")
```

Reply:
Done